ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -24,84 +24,93 @@
 {%                 endif %}
 {%             endfor %}
     {{ vrf.name }}:
+{%             if vrf.name == 'default' %}
+{%                 set default_vrf.enabled = true %}
+{%                 do route_targets.export.evpn.append('route-map RM-TENANT-NETWORK-DEFAULT-VRF') %}
+      rd: "{{ network_services_data.rd_admin_subfield }}:{{ bgp_vrf_id }}"
+      route_targets:
+        import: {{ route_targets.import | to_json }}
+        export: {{ route_targets.export | to_json }}
+{%             else %}
```

Comment:
if we exclude all the following knobs from the default vrf case, we should note that in the documentation. We can always add them one by one later.